{
  try {
    log.logMessage(msg,exception,true);
    return exception;
  }
 catch (  Throwable t) {
    return exception;
  }
}
